scrapy/extensions/telnet.py

```diff
@@ -50,8 +52,14 @@ def __init__(self, crawler):
         self.noisy = False
         self.portrange = [int(x) for x in crawler.settings.getlist('TELNETCONSOLE_PORT')]
         self.host = crawler.settings['TELNETCONSOLE_HOST']
+
```

Comment:
picky sorry, but any reason for the extra line?

Reply:
None, likely my brain separating things